atomic_reactor/plugins/pre_add_labels_in_df.py

```diff
@@ -67,24 +67,28 @@ class AddLabelsPlugin(PreBuildPlugin):
     }
 
     def __init__(self, tasker, workflow, labels,
-                 dont_overwrite=("Architecture", "architecture", ),
+                 dont_overwrite=("Architecture", "architecture"),
                  auto_labels=("build-date",
                               "architecture",
                               "vcs-type",
                               "vcs-ref",
                               "com.redhat.build-host"),
-                 aliases=None):
+                 aliases=None,
+                 dont_overwrite_if_in_dockerfile=("distribution-scope",)):
         """
         constructor
 
         :param tasker: DockerTasker instance
         :param workflow: DockerBuildWorkflow instance
         :param labels: dict, key value pairs to set as labels; or str, JSON-encoded dict
         :param dont_overwrite: iterable, list of label keys which should not be overwritten
+                               IF they are present in baseimage!
```

Comment:
Let's call it the "parent image" in the docstring, despite what the variables are called in the implementation.